### PR TITLE
Added maximum number of subs filtering

### PR DESCRIPTION
**Pull request by AlexKolpa**
@alexwalterbos

### CODE REVIEW

File: transformer.py [hunk 1]
```diff
@@ -50,6 +50,22 @@ def build_nodes(edge_list):
 		
 	return node_dict.values()
 
+def trim_to_size(edge_list, max_subs):
```
Comment (alexwalterbos) [inline transformer.py:54]: Could you add some comments to this method?

File: transformer.py [hunk 2]
```diff
@@ -88,6 +104,10 @@ def construct_graph(file_name, filter_min):
 		print 'filtering results below ' + str(filter_min)
 		edge_list = [edge for edge in edge_list if edge['value'] >= filter_min]
 
+	if max_subs > 0:
+		print 'removing subs to max of ' + str(max_subs)
```
Comment (alexwalterbos) [inline transformer.py]: "Limiting amount of subs to .." rolls more nicely.

**Comment by alexwalterbos** [comment]
I think the documentation would improve this for the teachers. :+1: after that.

**Comment by AlexKolpa** [comment]
@alexwalterbos fixed remarks

**Pull request merged**